addons/sale_timesheet/models/sale_order.py

```diff
@@ -12,31 +12,28 @@
 class SaleOrder(models.Model):
     _inherit = 'sale.order'
 
-    timesheet_ids = fields.Many2many('account.analytic.line', compute='_compute_timesheet_ids', string='Timesheet activities associated to this sale')
-    timesheet_count = fields.Float(string='Timesheet activities', compute='_compute_timesheet_ids', groups="hr_timesheet.group_hr_timesheet_user")
+    timesheet_ids = fields.Many2many('account.analytic.line', compute='_compute_timesheet_ids', string='Timesheet activities associated to this sale', compute_sudo=True)
```

Comment:
I am not sure if it is a good idea to compute that in sudo. 🤔

Reply:
No, so removed that field :slightly_smiling_face: